frontend/src/scenes/funnels/funnelLogic.ts

```diff
@@ -1129,19 +1128,42 @@ export const funnelLogic = kea<funnelLogicType>({
                 funnelStep: stepNumber,
             })
         },
-        openCorrelationPersonsModal: ({ entity, converted, resultType }) => {
-            personsModalLogic.actions.loadPeople({
-                action: { id: entity.id, name: entity.name, properties: entity.properties, type: entity.type },
-                label: entity.id,
-                date_from: '',
-                date_to: '',
-                filters: {
-                    ...values.filters,
-                    funnel_correlation_person_converted: converted ? 'true' : 'false',
-                    funnel_correlation_person_entity: entity,
-                },
-            })
-            eventUsageLogic.actions.reportCorrelationInteraction(resultType, 'person modal', { ...entity, converted })
+        openCorrelationPersonsModal: ({ correlation, success }) => {
+            if (correlation.result_type === FunnelCorrelationResultsType.Properties) {
+                const { breakdown, breakdown_value } = parseBreakdownValue(correlation.event.event)
+
+                personsModalLogic.actions.loadPeopleFromUrl({
+                    url: success ? correlation.success_people_url : correlation.failure_people_url,
+                    // I'm not sure why this is -2, I'm just copying from what was here previously
+                    stepNumber: success ? values.stepsWithCount.length : -2,
+                    label: breakdown,
+                    breakdown,
+                    breakdown_value,
+                })
+
+                eventUsageLogic.actions.reportCorrelationInteraction(
+                    FunnelCorrelationResultsType.Properties,
+                    'person modal',
+                    values.filters.funnel_correlation_person_entity
+                )
+            } else {
+                const { name, properties } = parseEventAndProperty(correlation.event)
+
+                personsModalLogic.actions.loadPeopleFromUrl({
+                    url: success ? correlation.success_people_url : correlation.failure_people_url,
+                    stepNumber: success ? values.stepsWithCount.length : -2,
+                    label: name,
+                    breakdown: '',
```

Comment:
StepNumber, and breakdown don't make sense here anymore?

Reply:
I think stepNumber is still needed to have the right modal title, although I'm just about to test this.

The breakdown and breakdown_value yup I guess shouldn't need to set that here, I'll make them optional.

Reply:
Simplified a little, but yeah, need this for display.

Can clean up in another PR